### DISCUSSION TITLE
Does it run on Windows 10 ?

### DISCUSSION THREAD

**Discussion by Ahazu77** [MicaForEveryone/MicaForEveryone#110] [Q&A] [answered]
May be a stupid, non-sense question, but does MicaForEveryone run on windows 10?

**Comment by xmine64** [comment, marked as answer]
Runs, but doesn't work properly and no backdrop customization, see #65

**Comment by bbmaster123** [comment]
still, I've been using it for several months to force one specific app follow titlebar accent color instead of just plain white. As long as that's all you care about it works fine on 10 right now, although no further customization as stated.
cheers

**Comment by Ahazu77** [comment]
Thanks for the reply bois =)
